docs/scripts/activemq/activemq.xml

```diff
@@ -146,6 +146,16 @@
               </compositeQueue>
             </virtualDestinations>
           </virtualDestinationInterceptor>
+          <virtualDestinationInterceptor>
```

Comment:
Just a minor thing but can you add a section to the update docs that the activemq configuration needs to be update?

Reply:
sure, done